### PR TITLE
Increase line height of list items

### PR DESCRIPTION
**Pull request by cedricium**
Fix for #569. How does the change look @ryanfeeley?

**Before:**
screen shot 2018-01-15 at 1 42 13 am

**After:**
screen shot 2018-01-15 at 1 41 40 am

### CODE REVIEW

File: src/sidebar/styles.css [hunk 1]
```diff
@@ -50,6 +50,10 @@ body .ck-editor .ck-editor__editable {
   padding-left: 20px;
 }
 
+.ck-editor__editable ol > li, .ck-editor__editable ul > li {
+  line-height: 1.5rem;
```
Comment (Natim) [inline src/sidebar/styles.css]: Is 1rem colliding?

Comment (cedricium) [reply src/sidebar/styles.css]: @Natim it is. 1rem is 16px which I think matches the current P font-size, thus the reason for the overlapping.

Comment (Natim) [reply src/sidebar/styles.css]: Ok thanks !

**Comment by ryanfeeley** [comment]
@cedricium Looks good, although my example was using code formatting.

- `I wonder`
- `how GitHub’s`
- `text editor`
- `does line-height`
- ` as code`
- ok, good I'm not crazy
- i think they have a nice line height

**Pull request merged**